From 560d491401a62d759f4b013f120c8d4cad18523c Mon Sep 17 00:00:00 2001
From: Arie Brown <abrown@habana.ai>
Date: Mon, 22 Nov 2021 11:23:02 +0200
Subject: [PATCH] [SW-0] gaudi2: arm: temporary inc bin max size

For overcome arm preboot bin size issue we temporary remove 1KB
safety buffer and use the whole 64KB.
After a long term solution will be submitted, the limit of max size
should be increased to 127KB.

Change-Id: I46109769fc527f3d3ff0ea9969dec89b0a4cb2c7
Signed-off-by: Arie Brown <abrown@habana.ai>
(cherry picked from commit 64653d9112f4e55965c33e8f5c93920e5326ffd0)
---

diff --git a/arch/gaudi2/arm/Makefile b/arch/gaudi2/arm/Makefile
index 6778b88..60c2025 100644
--- a/arch/gaudi2/arm/Makefile
+++ b/arch/gaudi2/arm/Makefile
@@ -2,7 +2,7 @@
 
 FW_SIZE := 0x5400
 PPBOOT_MAX_SIZE := 20480
-PREBOOT_MAX_SIZE := 64512
+PREBOOT_MAX_SIZE := 65536
 
 PREBOOT_SRCS_C += $(shell ls $(PREBOOT_ROOT_DIR)/lib/preboot/thermal_v2.c)
 
